### PR TITLE
feat(client): Admin page for user to accept account requests

### PR DESCRIPTION
**Pull request by sujvn**
Admin page shows list of accounts that were requested
User can accept or deny accounts that were requested
If accepted, user that requested account will receive email

### CODE REVIEW

**Comment by Mackbellemore** [comment]
Task linked: [CU-mh216d Admin page to accept account creation requests](https://app.clickup.com/t/mh216d)

File: client/src/pages/index.jsx [hunk 1]
```diff
@@ -34,17 +39,23 @@ const Index = observer(() => {
...
           </Switch>
         </main>
       </BrowserRouter>
     </>
   );
-});
+};
 
-export default Index;
+export default observer(Index);
```
Comment (Mackbellemore) [inline client/src/pages/index.jsx:61]: ''

File: client/src/components/RegisterUserModal.jsx [hunk 2]
```diff
@@ -83,11 +83,11 @@ const RegisterUserModal = () => {
     }
   };
 
-  // Validates Username
+  // Validates email
   const registerHandleEmailValidation = (e) => {
     const email = e.target.value;
-    // any number of characters followed by @ followed by any number of characters followed by a . then any number of characters
-    const regex = /^[a-zA-Z0-9]+@(?:[a-zA-Z0-9]+\.)+[A-Za-z]+$/;
+    const regex = /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
```
Comment (Mackbellemore) [inline client/src/components/RegisterUserModal.jsx:89]: Dear mr sujan and gordon, great work on the regex

**Review by KarinKazarian — APPROVED** [review]
Great job guys!

**Review by Mackbellemore — APPROVED** [review]
:o:

**Pull request merged**